charts/gardener-extension-shoot-oidc-service/templates/service.yaml

```diff
@@ -8,8 +8,13 @@ kind: Service
 metadata:
   name: gardener-extension-{{ .Values.serviceName }}
   namespace: {{ .Release.Namespace }}
-{{- if .Values.ignoreResources }}
   annotations:
+    networking.resources.gardener.cloud/from-world-to-ports: '[{"protocol":"TCP","port":{{ .Values.webhookConfig.serverPort }}}]'
```

Comment:
Is this line needed because the extension has a mutating webhook?

Reply:
Yes, exactly. The API server of the runtime cluster must be allowed to reach the webhook port.